crates/cashu/src/nuts/nut11.rs

```diff
@@ -215,7 +214,7 @@ impl TryFrom<P2PKConditions> for Secret {
             return Err(Error::Amount);
         }
 
-        let data: PublicKey = pubkeys[0].clone().into();
+        let data = pubkeys[0].clone();
 
         let data = data.to_string();
```

Comment:
Throws a runtime error if no item at index 0. Might want to do something like

```suggestion
        let data = match pubkeys.get(0) {
            Some(data) => data.to_string(),
            None => return Err(Error::CustomError(String::from("No pubkey at index 0"))),
        };
```

Reply:
There is a check that there is at least one pubkey on the line above, but this is a cleaner way to do it thanks